src/saga/Registrationsaga.js

```diff
@@ -0,0 +1,37 @@
+import { takeLatest, put, call } from "redux-saga/effects";
+import * as actions from "../Redux/actionTypes/actionsTypes";
+
+import {
+  RegistationSuccess,
+  RegistationError,
+} from "../Redux/createAction/createAction";
+import axios from "axios";
+
+function* sagaregisRequest(action) {
+  try {
+    let username = action.payload.username;
+    let password = action.payload.password;
+    let option = action.payload.option;
+    console.log(username, password, option);
```

Comment:
always never console log in pull request

Reply:
_resolved_